src/test/java/seedu/recipe/storage/jsonadapters/JsonAdaptedPortionUnitTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.recipe.storage.jsonadapters;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.recipe.commons.exceptions.IllegalValueException;
+import seedu.recipe.model.recipe.unit.PortionUnit;
+
+//@@author alson001
+public class JsonAdaptedPortionUnitTest {
+    private static final String VALID_UNIT = "grams";
+    private static final String INVALID_UNIT = "";
+    private static final String NULL_UNIT = null;
```

Comment:
null is tested in jsonadapted portion but not the rest, can standardise further

Reply:
standardised